src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -287,9 +299,9 @@ object UserService {
     email.value match {
       case emailString if blockedEmailDomains.exists(domain => {
         emailString.endsWith("@" + domain) || emailString.endsWith("." + domain)
-      }) => IO.raiseError(new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.BadRequest, s"email domain not permitted [${email.value}]")))
+      }) => IO.raiseError(new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.BadRequest, s"email domain not permitted")))
```

Comment:
Again, appreciate the effort to try to reduce email addresses in logs, but I am worried about the usefulness of this message without the email.  Can we at least log the `domain`?

Reply:
I would say just leave email addresses in.